### PR TITLE
[MeshingApplication] Adding EnsureProperties method

### PR DESCRIPTION
**Pull request by loumalouomega**
This ensures that the properties of the elements and conditions are defined on the model part

### CODE REVIEW

File: applications/MeshingApplication/custom_utilities/meshing_utilities.cpp [hunk 1]
```diff
@@ -21,6 +22,65 @@ namespace Kratos
 {
 namespace MeshingUtilities
 {
+void EnsureProperties(ModelPart& rModelPart)
```
Comment (philbucher) [inline applications/MeshingApplication/custom_utilities/meshing_utilities.cpp]: to me the name is not clear what it means
But I also don't have a better suggestion :)

File: applications/MeshingApplication/custom_python/add_custom_utilities_to_python.cpp [hunk 2]
```diff
@@ -50,9 +50,10 @@ namespace py = pybind11;
 void AddCustomUtilitiesToPython(pybind11::module& m)
 {
     // MeshingUtilities
+    m.def("EnsureProperties", &MeshingUtilities::EnsureProperties);
```
Comment (philbucher) [inline applications/MeshingApplication/custom_python/add_custom_utilities_to_python.cpp]: you could also expose the utils as a submodule, I recently figured it out and think that it is a good solution: https://github.com/KratosMultiphysics/Kratos/commit/ff553718c1b9d5c981e141219ac93033ecff44ce#diff-3d608579b83d29266b38af995ee7c6d4

Comment (loumalouomega) [reply applications/MeshingApplication/custom_python/add_custom_utilities_to_python.cpp]: OK, I will check

File: applications/MeshingApplication/custom_utilities/meshing_utilities.cpp (hunk 1 above)
Comment (loumalouomega) [reply applications/MeshingApplication/custom_utilities/meshing_utilities.cpp]: EnsureModelPartOwnsProperties?

Comment (philbucher) [reply applications/MeshingApplication/custom_utilities/meshing_utilities.cpp]: `EnsureModelPartHasProperties` would be more aligned with the other names of the functions in the Modelpart

**Comment by loumalouomega** [comment]
Done @philbucher

File: applications/MeshingApplication/custom_utilities/meshing_utilities.cpp (hunk 1 above)
Comment (philbucher) [reply applications/MeshingApplication/custom_utilities/meshing_utilities.cpp]: will you change this?

**Comment by loumalouomega** [comment]
Ping @philbucher

**Review by philbucher — APPROVED** [review]
to me this looks ok, I don't know if someone that actually uses this wants to have a look

**Pull request merged**